common/src/main/java/com/codeforcommunity/auth/JWTHandler.java

```diff
@@ -57,7 +58,8 @@ private DecodedJWT getDecodedJWT(String jwt) throws JWTVerificationException {
   }
 
   private Date getTokenExpiration(boolean isRefresh) {
-    long exp = isRefresh ? AuthUtils.refresh_exp : AuthUtils.access_exp;
+    long exp = isRefresh ? Long.valueOf(PropertiesLoader.getExpirationProperties().getProperty("days_refresh_expiration"))
```

Comment:
Break up this logic into multiple lines please

Reply:
k

Reply:
it has been changed to a static var though